src/azure-cli-tests/azure/cli/tests/base.py

```diff
@@ -0,0 +1,167 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import unittest
+import os.path
+import inspect
+import vcr
+import subprocess
+import json
+import shlex
+
+from azure.cli.tests.patches import (patch_load_cached_subscriptions, patch_main_exception_handler,
+                                     patch_retrieve_token_for_user)
+from azure.cli.tests.exceptions import CliExecutionError
+from azure.cli.tests.const import ENV_LIVE_TEST, ENV_SKIP_ASSERT, MOCKED_SUBSCRIPTION_ID
+from azure.cli.tests.recording_processors import (SubscriptionRecordingProcessor,
+                                                  OAuthRequestResponsesFilter,
+                                                  GeneralNameReplacer)
+from azure.cli.tests.utilities import create_random_name
+
+
+class ScenarioTest(unittest.TestCase):
+    def __init__(self, method_name):
+        super(ScenarioTest, self).__init__(method_name)
```

Comment:
I use "debug=True" very frequently with the existing test base. How do I accomplish this now?